<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
	border-image:url(:/weaUI/weaUI.png);
	color:rgb(255,255,255);
}

QLabel{
	font: 25 10pt &quot;Microsoft YaHei&quot;;
	border-radius: 4px;
	background-color: argb(60, 60, 60, 130);
	color: rgb(255, 255, 255);
}

Line{
	background-color: rgb(0, 85, 0);
}</string>
    </property>
    <widget class="QLineEdit" name="cityEdit">
     <property name="geometry">
      <rect>
       <x>575</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>400</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>2022-06-03</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="hLine1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>780</width>
       <height>1</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="hLine11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>335</y>
       <width>351</width>
       <height>1</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="hLine10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>183</y>
       <width>351</width>
       <height>1</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>171</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;华文琥珀&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(85, 255, 0);</string>
     </property>
     <property name="text">
      <string>天气预报</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>351</width>
       <height>145</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>风向</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>风速</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>PM2.5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>湿度</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="pm25">
        <property name="text">
         <string>20</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="fengxiang">
        <property name="text">
         <string>东南风</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="fengli">
        <property name="text">
         <string>8级</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="shidu">
        <property name="text">
         <string>95%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>121</width>
       <height>37</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-color: argb(60, 60, 60, 0);
image: url(:/weatherIco/location.ico);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="cityName">
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Microsoft YaHei UI&quot;;
background-color: rgba(60, 60, 60, 0);</string>
        </property>
        <property name="text">
         <string>北京市</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>421</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="6" column="3">
       <widget class="QLabel" name="high3">
        <property name="text">
         <string>1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="weather5">
        <property name="text">
         <string>小雪</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="weather1">
        <property name="text">
         <string>晴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="quality4">
        <property name="text">
         <string>轻度污染</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="week0">
        <property name="text">
         <string>昨天</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" rowspan="2">
       <widget class="QLabel" name="weaIcon0">
        <property name="styleSheet">
         <string notr="true">image:url(:/day/day/中到大雨.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="high1">
        <property name="text">
         <string>1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="quality1">
        <property name="text">
         <string>良</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="day4">
        <property name="text">
         <string>05</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3" rowspan="2">
       <widget class="QLabel" name="weaIcon3">
        <property name="styleSheet">
         <string notr="true">image:url(:/day/day/中到大雨.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="5">
       <widget class="QLabel" name="low5">
        <property name="text">
         <string>-1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="day5">
        <property name="text">
         <string>06</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="low0">
        <property name="text">
         <string>-1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="day2">
        <property name="text">
         <string>03</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="quality0">
        <property name="text">
         <string>优</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2" rowspan="2">
       <widget class="QLabel" name="weaIcon2">
        <property name="styleSheet">
         <string notr="true">image:url(:/day/day/中到大雨.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="high4">
        <property name="text">
         <string>1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="weather0">
        <property name="text">
         <string>阴</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="weather4">
        <property name="text">
         <string>大雨</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="quality5">
        <property name="text">
         <string>轻度污染</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="week1">
        <property name="text">
         <string>今天</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QLabel" name="low4">
        <property name="text">
         <string>-1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" rowspan="2">
       <widget class="QLabel" name="weaIcon1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>65</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image:url(:/day/day/中到大雨.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="week2">
        <property name="text">
         <string>明天</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="weather2">
        <property name="text">
         <string>多云</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="day3">
        <property name="text">
         <string>04</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="high2">
        <property name="text">
         <string>1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="low2">
        <property name="text">
         <string>-1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="quality2">
        <property name="text">
         <string>轻度污染</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="week5">
        <property name="text">
         <string>周三</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="4" rowspan="2">
       <widget class="QLabel" name="weaIcon4">
        <property name="styleSheet">
         <string notr="true">image:url(:/day/day/中到大雨.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="week3">
        <property name="text">
         <string>周一</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="high0">
        <property name="text">
         <string>1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLabel" name="high5">
        <property name="text">
         <string>1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="weather3">
        <property name="text">
         <string>小雨</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0" rowspan="2" colspan="6">
       <widget class="QLabel" name="curveLb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>101</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="day1">
        <property name="text">
         <string>02</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="day0">
        <property name="text">
         <string>01</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="quality3">
        <property name="text">
         <string>轻度污染</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="low1">
        <property name="text">
         <string>-1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="5" rowspan="2">
       <widget class="QLabel" name="weaIcon5">
        <property name="styleSheet">
         <string notr="true">image:url(:/day/day/中到大雨.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="week4">
        <property name="text">
         <string>周二</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLabel" name="low3">
        <property name="text">
         <string>-1 ℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/weatherIco/leaf.ico);
background-color:argb(255,255,255,0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="qualityLb">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>52</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: argb(60, 60, 60, 0);
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>优</string>
     </property>
    </widget>
    <widget class="QLabel" name="temperature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>9℃</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="weather">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: argb(60, 60, 60, 0);
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>晴</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="weaIcon">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/day/day/晴.png);
background-color:rgba(60, 60, 60, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>201</width>
       <height>98</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;感&lt;br/&gt;冒&lt;br/&gt;指&lt;br/&gt;数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QTextBrowser" name="ganmaoBrowser">
        <property name="styleSheet">
         <string notr="true">background-color: argb(60, 60, 60, 130);
color: rgb(255, 255, 255);
border:0</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;极少数敏感人群应该减少户外活动&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>345</y>
       <width>1</width>
       <height>100</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="searchBt">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>13</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/weatherIco/search.ico)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="refreshBt">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/weatherIco/refresh.ico)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sunRaiseSet">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>350</y>
       <width>140</width>
       <height>96</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="notice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>记得随身携带雨伞哦</string>
     </property>
    </widget>
    <zorder>temperature</zorder>
    <zorder>cityEdit</zorder>
    <zorder>dateLabel</zorder>
    <zorder>hLine1</zorder>
    <zorder>hLine11</zorder>
    <zorder>hLine10</zorder>
    <zorder>label</zorder>
    <zorder>gridLayoutWidget</zorder>
    <zorder>gridLayoutWidget_2</zorder>
    <zorder>gridLayoutWidget_3</zorder>
    <zorder>label_7</zorder>
    <zorder>weather</zorder>
    <zorder>weaIcon</zorder>
    <zorder>gridLayoutWidget_4</zorder>
    <zorder>line</zorder>
    <zorder>searchBt</zorder>
    <zorder>refreshBt</zorder>
    <zorder>sunRaiseSet</zorder>
    <zorder>qualityLb</zorder>
    <zorder>notice</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
